### PR TITLE
[Generator | Base] - Upgrade 0.28

### PR DESCRIPTION
**Pull request by GantMan**
#### Ignite Base Change
- [x] Works on iOS/Android/XDE
- [x] Tests Pass (run: `npm run test`)
- [x] Code is StandardJS compliant (run: `npm run lint`)

#### Generator Change
- [x] package.json lists all generators under "files"
- [x] I have run and tested `npm compile` with latest src

@skellock  - plz review

### CODE REVIEW

File: ignite-base/android/app/build.gradle [hunk 1]
```diff
@@ -78,12 +84,12 @@ def enableProguardInReleaseBuilds = false
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    buildToolsVersion "23.0.1"
```
Comment (skellock) [inline ignite-base/android/app/build.gradle]: Not a huge deal, but 23.0.3 is the latest in that branch.   I'm guessing you didn't change these manually though.

File: ignite-base/android/app/build.gradle [hunk 2]
```diff
@@ -78,12 +84,12 @@ def enableProguardInReleaseBuilds = false
...
 
     defaultConfig {
         applicationId "com.rnbase"
         minSdkVersion 16
-        targetSdkVersion 23
+        targetSdkVersion 22
```
Comment (skellock) [inline ignite-base/android/app/build.gradle]: Wonder why this changed too?

**Comment by kevinvangelder** [comment]
Bad @GantMan! `Base package.json does not match package.json.template`

File: ignite-base/android/app/build.gradle (hunk 2 above)
Comment (GantMan) [inline ignite-base/android/app/build.gradle]: Yeah, this is what comes down in latest react native *shrug*

File: ignite-base/package.json [hunk 3]
```diff
@@ -20,16 +20,17 @@
   },
   "dependencies": {
     "apisauce": "^0.1.5",
+    "chai-immutable": "^1.6.0",
```
Comment (skellock) [inline ignite-base/package.json]: Don't think we need this do we?

**Comment by skellock** [comment]
Looks good.  The whole indentation changes in Info.plist threw me for a loop for a sec, but I'm sure it's just that tool you're using reformatting things.

:shipit:

**Comment by GantMan** [comment]
@skellock  - fixed indentation, it bugged me, too.

**Comment by skellock** [comment]
Its a trigger for me because of my .NET days.  Sometimes the act of open up a project in Visual Studio would change the source file.   /cut /cut /tears

**Pull request merged**